src/AppBundle/Entity/Administrator.php

```diff
@@ -44,6 +45,13 @@ class Administrator implements UserInterface
     private $password;
 
     /**
+     * @var string|null
+     *
+     * @ORM\Column(nullable=true)
+     */
+    private $googleAuthenticatorSecret;
```

Comment:
This is not the code, this is the secret used to generate the code. A bit like a private key. It's stored on the server and on the application only once.